language/move-prover/bytecode-to-boogie/src/translator.rs

```diff
@@ -1263,3 +1273,11 @@ pub fn get_field_infos(
             .collect()
     }
 }
+
+/// Get return types of function via definition index.
+pub fn get_return_types(module: &VerifiedModule, idx: usize) -> &[SignatureToken] {
```

Comment:
This looks like a useful function! Might make sense to move it to `impl CompiledModule`?

Reply:
There is like a dozen more of those, and they are spread quite unsystematically over the code. I have a TODO in translator.rs to consolidate all this, and it is perhaps a good idea to consolidate this with the core data structures. My understanding is it would belong into views.rs.